<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GuanxingBox</class>
 <widget class="QWidget" name="GuanxingBox">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>924</width>
    <height>440</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>20</number>
   </property>
   <property name="topMargin">
    <number>50</number>
   </property>
   <property name="rightMargin">
    <number>20</number>
   </property>
   <property name="bottomMargin">
    <number>30</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QFrame" name="leftFrame">
       <property name="minimumSize">
        <size>
         <width>214</width>
         <height>276</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>214</width>
         <height>276</height>
        </size>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="margin">
         <number>0</number>
        </property>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <property name="spacing">
           <number>0</number>
          </property>
          <item>
           <spacer name="verticalSpacer_8">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <spacer name="horizontalSpacer_3">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLabel" name="messageLabel">
              <property name="text">
               <string>&lt;font color=white&gt;&lt;b&gt;观星&lt;/b&gt;&lt;font&gt;</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_4">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <spacer name="horizontalSpacer_5">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="sgsui::ResultPlayerLabel" name="zhugeliangLabel"/>
            </item>
            <item>
             <spacer name="horizontalSpacer_6">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <spacer name="verticalSpacer_7">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <property name="spacing">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="upHint1">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>40</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>&lt;font color=white&gt;&lt;b&gt;上&lt;/b&gt;&lt;font&gt;</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="upHint2">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>40</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>&lt;font color=white&gt;&lt;b&gt;上&lt;/b&gt;&lt;font&gt;</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="middleFrame">
       <property name="minimumSize">
        <size>
         <width>503</width>
         <height>276</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>503</width>
         <height>276</height>
        </size>
       </property>
       <widget class="QFrame" name="upperCardRect1">
        <property name="geometry">
         <rect>
          <x>1</x>
          <y>1</y>
          <width>93</width>
          <height>130</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>93</width>
          <height>130</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>93</width>
          <height>130</height>
         </size>
        </property>
       </widget>
       <widget class="QFrame" name="upperCardRect2">
        <property name="geometry">
         <rect>
          <x>103</x>
          <y>1</y>
          <width>93</width>
          <height>130</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>93</width>
          <height>130</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>93</width>
          <height>130</height>
         </size>
        </property>
       </widget>
       <widget class="QFrame" name="upperCardRect3">
        <property name="geometry">
         <rect>
          <x>205</x>
          <y>1</y>
          <width>93</width>
          <height>130</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>93</width>
          <height>130</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>93</width>
          <height>130</height>
         </size>
        </property>
       </widget>
       <widget class="QFrame" name="upperCardRect4">
        <property name="geometry">
         <rect>
          <x>307</x>
          <y>1</y>
          <width>93</width>
          <height>130</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>93</width>
          <height>130</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>93</width>
          <height>130</height>
         </size>
        </property>
       </widget>
       <widget class="QFrame" name="upperCardRect5">
        <property name="geometry">
         <rect>
          <x>409</x>
          <y>1</y>
          <width>93</width>
          <height>130</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>93</width>
          <height>130</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>93</width>
          <height>130</height>
         </size>
        </property>
       </widget>
       <widget class="QFrame" name="lowerCardRect1">
        <property name="geometry">
         <rect>
          <x>1</x>
          <y>145</y>
          <width>93</width>
          <height>130</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>93</width>
          <height>130</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>93</width>
          <height>130</height>
         </size>
        </property>
       </widget>
       <widget class="QFrame" name="lowerCardRect2">
        <property name="geometry">
         <rect>
          <x>103</x>
          <y>145</y>
          <width>93</width>
          <height>130</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>93</width>
          <height>130</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>93</width>
          <height>130</height>
         </size>
        </property>
       </widget>
       <widget class="QFrame" name="lowerCardRect3">
        <property name="geometry">
         <rect>
          <x>205</x>
          <y>145</y>
          <width>93</width>
          <height>130</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>93</width>
          <height>130</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>93</width>
          <height>130</height>
         </size>
        </property>
       </widget>
       <widget class="QFrame" name="lowerCardRect4">
        <property name="geometry">
         <rect>
          <x>307</x>
          <y>145</y>
          <width>93</width>
          <height>130</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>93</width>
          <height>130</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>93</width>
          <height>130</height>
         </size>
        </property>
       </widget>
       <widget class="QFrame" name="lowerCardRect5">
        <property name="geometry">
         <rect>
          <x>409</x>
          <y>145</y>
          <width>93</width>
          <height>130</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>93</width>
          <height>130</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>93</width>
          <height>130</height>
         </size>
        </property>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="rightFrame">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>20</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <layout class="QVBoxLayout" name="_3">
          <property name="spacing">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="downHint1">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>40</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>40</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>&lt;font color=white&gt;&lt;b&gt;下&lt;/b&gt;&lt;font&gt;</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="downHint2">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>40</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>40</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>&lt;font color=white&gt;&lt;b&gt;下&lt;/b&gt;&lt;font&gt;</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="horizontalSpacer_7">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QVBoxLayout" name="_2">
          <item>
           <spacer name="verticalSpacer_3">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>58</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="hintLabel1">
            <property name="text">
             <string>&lt;font color=white&gt;牌堆顶部&lt;font&gt;</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_4">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>57</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <spacer name="verticalSpacer_6">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>58</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="hintLabel2">
            <property name="text">
             <string>&lt;font color=white&gt;牌堆底部&lt;font&gt;</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_5">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>57</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="leftMargin">
      <number>50</number>
     </property>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="sgsui::GameButton" name="okButton"/>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>sgsui::GameButton</class>
   <extends>QLabel</extends>
   <header>sgsui_GameButton.h</header>
  </customwidget>
  <customwidget>
   <class>sgsui::ResultPlayerLabel</class>
   <extends>QLabel</extends>
   <header>sgsui_ResultPlayerLabel.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
